src/hooks/useStore.ts

```diff
@@ -1,16 +1,16 @@
 import useSWR from "swr"
 import { useFetcher } from "./useFetcher"
-import type { Character, Store } from "../types"
+import type { Character, Store, StoreType } from "../types"
 import { useEffect } from "react"
 
-let key = (char?: Character) =>
+let key = (char: Character | undefined, storeType: StoreType) =>
```

Comment:
Is there a reason for changing `char?: Character` to `char: Charatcer | undefined`. AFAIK they are pretty much the same (optional params will take undefined as a valid value)

Reply:
A required parameter can't follow an optional one, so it was either explicitly say char can be undefined, or also make storeType optional (which doesn't make any sense - it's got a default value of credits and can't ever be undefined).

I felt this was the nicer way of the two, but am open to changing it. As you say, they're literally equivalent, but T|undefined has the advantage of a more consistent call signature.